### PR TITLE
Product and service

### PR DESCRIPTION
**Pull request by jdaviscooke**
Add product pages 
Add marketing copy to product pages
Use Feature Row for the consulting landing page

### CODE REVIEW

File: _consulting/architecting.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Cloud Architecting
+layout: splash
+permalink: /consulting/architecting
+
+header:
+  overlay_image: /assets/images/website/architecting.jpg
+  # image: /assets/images/website/architecting.jpg
+  teaser: /assets/images/website/architecting.jpg
```
Comment (am17torres) [inline _consulting/architecting.markdown]: this behaves differently than the one above. Is there a preferred use of image, teaser, and/or overlay?

File: _consulting/experience.markdown [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: Developer Experience
+layout: splash
+permalink: /consulting/experience
+
+header:
+  overlay_image: /assets/images/website/data.jpg
+  teaser: /assets/images/website/data.jpg
```
Comment (am17torres) [inline _consulting/experience.markdown]: this one doesn't have image

File: _includes/contact_box [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<div>
```
Comment (am17torres) [inline _includes/contact_box:1]: slick

File: _pages/consulting.markdown [hunk 4]
```diff
@@ -0,0 +1,89 @@
+---
+title: BitHippie Tech Consulting Services 👋
+layout: splash
+permalink: /consulting/
+collection: consulting
+# entries_layout: grid
+# classes: wide
```
Comment (am17torres) [inline _pages/consulting.markdown]: kill it

File: _pages/consulting.markdown [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    url: "/consulting/engineering"
+    btn_label: "Learn More"
+    btn_class: "btn--inverse"
+
+---
+
+
+{% include feature_row %}
+
+<!--
```
Comment (am17torres) [inline _pages/consulting.markdown]: kill it?

File: assets/css/main.scss [hunk 6]
```diff
@@ -70,6 +70,70 @@ $bh_brown: #a65a36 !default;
     }
 }
 
+.page__hero--overlay .page__lead {
```
Comment (am17torres) [inline assets/css/main.scss:73]: can you comment any changes you made so I know why

**Review by am17torres — APPROVED** [review]
Nothing blocking. :shipit:

File: assets/css/main.scss (hunk 6 above)
Comment (jdaviscooke) [reply assets/css/main.scss:73]: I don't know where this came from 🤔 
I did do an upgrade of some packages to get around an issue where liquid wouldn't work with jekyll on the version of ruby i was running, and the version of ruby that was suggested didn't work on Apple silicone. I wonder if this got updated as a part of new package versions?

**Pull request merged**